### PR TITLE
Add gold tests for Wool and OpenMP.

### PR DESCRIPTION
**Pull request by kffaxen**

### CODE REVIEW

File: tests/RegressionTests.hs [hunk 1]
```diff
@@ -309,6 +312,10 @@ nativeOpts :: Options
 nativeOpts = defaultOptions{useNativeArrays=True}
 nativeRetOpts :: Options
 nativeRetOpts = defaultOptions{useNativeReturns=True}
+woolOptions :: Options
+woolOptions = sicsOptions3
+openMPOptions :: Options
+openMPOptions = defaultOptions{platform = c99OpenMp}
```
Comment (pjonsson) [inline tests/RegressionTests.hs:318]: Do openMPOpts = c99OpenMpPlatformOptions so we know we aren't redefining the content of the options.

File: tests/RegressionTests.hs [hunk 2]
```diff
@@ -309,6 +312,10 @@ nativeOpts :: Options
 nativeOpts = defaultOptions{useNativeArrays=True}
 nativeRetOpts :: Options
 nativeRetOpts = defaultOptions{useNativeReturns=True}
+woolOptions :: Options
```
Comment (pjonsson) [inline tests/RegressionTests.hs:315]: The naming convention for variables for Options is "Opts" in this part of the code.

File: tests/RegressionTests.hs (hunk 1 above)
Comment (kffaxen) [reply tests/RegressionTests.hs:318]: Ok, missed that binding!

File: tests/RegressionTests.hs (hunk 2 above)
Comment (kffaxen) [reply tests/RegressionTests.hs:315]: Ok.

**Pull request merged**